lesson5/lesson5.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
add .iml files (and other not related to code) to .gitignore

Reply:
I just don't understand, what we want here, and what don't.

Reply:
in repository should be only project files, but not generated by idea.